api/python/t4/api.py

```diff
@@ -14,13 +14,39 @@
 
 
 def copy(src, dest, meta=None):
+    """
+    Retrieves ``src`` object from T4, and writes that to to the ``dest``
```

Comment:
Maybe "Copies `src` object from T4 to `dest`" for brevity. The retrieves and writes construction of the sentence can be in the paragraph description